tests/Dockerfile.hepcrawl_py3

```diff
@@ -0,0 +1,11 @@
+FROM python:3-slim
+
+WORKDIR /usr/src/app
+
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y build-essential \
+  && rm -rf /var/lib/apt/lists
+
+COPY . .
```

Comment:
Since you are copying everything, you might want to define a `.dockerignore` file avoiding copying tests and other useless files.